docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -568,6 +568,7 @@ testImplementation("io.rest-assured:rest-assured")
 testImplementation("io.quarkus:quarkus-junit5")
 ----
 
+ifndef::no-deprecated-test-resource[]
```

Comment:
@rolfedh I don't think either Wiremock or oidc test client should be labeled as `deprecated-test-resource`, they should not be really conditionalized, only `KeycloakTestResourceLifecycleManager` should be

Reply:
@rolfedh Thanks, only `Using KeycloakTestResourceLifecycleManager` should be marked as a deprecated test resource, `Dev Services for Keycloak` is our recommended test approach :-)